app/templates/events/view/sessions.hbs

```diff
@@ -0,0 +1,103 @@
+<div class="ui basic buttons">
+  <button class="ui button">All</button>
+  <button class="ui button">Pending</button>
+  <button class="ui button">Accepted</button>
+  <button class="ui button">Confirmed</button>
+  <button class="ui button">Rejected</button>
+</div>
+<button class="ui teal button">Create Session</button>
+<select class="ui dropdown">
```

Comment:
@niranjan94 Should I add a dropdown icon as well with this or only text `Export`?